examples/scikit_cpu.py

```diff
@@ -0,0 +1,46 @@
+from datetime import timedelta
```

Comment:
since the other examples has the name of the dataset in the file name, maybe its better to name this one breast_cancer_cpu ?